CDEFramework/Libraries/LegacyDictionarySupport/BusinessObjects/GlossaryTerm.cs

```diff
@@ -0,0 +1,367 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace LegacyDictionarySupport
+{
+    /// <summary>
+    /// Represents a Glossary Term as returned by http://webapis.cancer.gov/glossary/v1/
+    /// </summary>
+    public class GlossaryTerm
+    {
+
+        #region Subclasses of this object
```

Comment:
Just to be clear, I don't have an issue with nesting classes like this. It makes sense.  Just a general note though that these really shouldn't be called "subclasses."  The term "subclass" means that the class extends the functionality its superclass (the class which is being extended).

These are really "Classes for child objects."

**IFF** there are other changes for this PR, please update the comment.